README.md

```diff
@@ -50,7 +50,22 @@ if (device) {
 }
 ```
 
-If you're a *huge* fan of Litra devices and you have multiple plugged in at the same time, it'll return whatever one it happens to find first.
+If you're a *huge* fan of Litra devices and you have multiple plugged in at the same time, use `findDevices` instead:
+
+```js
+const devices = findDevices();
+
+if (devices.length > 0) {
+  console.log(`Found ${devices.length} devices connected`);
+  for (let i = 0; i < devices.length; ++i) {
+    console.log(`Device ${i + 1}: ${devices[i].type}`);
```

Comment:
Is there any reasonable way that a user can differentiate which device is which? At the minimum they can see the `type` if they have a Glow and a Beam - but it's not great fidelity!

Reply:
We could add `serialNumber` as that will be unique to the device.

For each device from HID we have:

```json
{
  vendorId: 1133,
  productId: 51457,
  path: '...',
  serialNumber: '...',
  manufacturer: 'Logitech',
  product: 'Litra Beam',
  release: 256,
  interface: 0,
  usagePage: 65347,
  usage: 514
}
```

So, `Device` could be updated to:

```ts
// Conforms to the interface of `node-hid`'s `HID.HID`. Useful for mocking.
export interface Device {
  hid: {
    write: (values: number[] | Buffer) => number;
  };
  type: DeviceType;
  serialNumber: string;
}
```

Reply:
That sounds like a great change. Are you seeing different serial numbers for each of your devices? Perhaps you wouldn't mind sharing them with me so I can check that they are different to mine?

A serial number isn't as good as "what USB port is it plugged into?" - but at least it allows you to deterministically know which devices is which.

Reply:
In other words, please do go ahead and add that to `Device` - that would be excellent.

Reply:
Ok, pushed https://github.com/timrogers/litra/pull/86/commits/0ed3c3e8f063e99e3665d4c8fa627de82ed2f63a

I've tested it over in the examples branch I've created, and it lists different serial numbers for each of my devices. They should all be unique.

Since I used the GitHub UI to accept the recommendations you made (and didn't update the commit message), the commit history here has become a bit of a mess. Do you want me to sort that out (I'll probably create a fresh PR), or are you going to squash this Squash and Merge?